jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -279,11 +279,11 @@ static void initCompressionFlag() {
 	 * Latin1 string.
 	 *
 	 * <p>This API implicitly assumes the following:
-	 * <blockquote><pre>
-	 *     - {@code length >= 0}
-	 *     - {@code start >= 0}
-	 *     - {@code start + length <= data.length}
-	 * <blockquote><pre>
+	 * <ul>
+	 * <li>{@code length >= 0}</li>
+	 * <li>{@code start >= 0}</li>
+	 * <li>{@code start + length <= data.length}</li>
```

Comment:
`data` should be `c` (in both copies of this method)

Reply:
Updated for both classes.